autorest/code_generator.py

```diff
@@ -66,13 +65,14 @@ def _build_exceptions_set(self, yaml_data):
                 for exception in operation['exceptions']:
                     if not exception.get('schema'):
                         continue
-                    exceptions_set.add(exception['schema']['language']['default']['name'])
+                    exceptions_set.add(exception['schema']['language']['python']['name'])
         return exceptions_set
 
     def _create_code_model(self, yaml_code_model):
+        NameConverter().convert_yaml_names(yaml_code_model)
         # Create a code model
         code_model = CodeModel()
-        code_model.client_name = yaml_code_model['info']['title'].replace(" ", "")  # FIXME do a proper namer
+        code_model.python_client_name = yaml_code_model['info']['python_title']  # FIXME do a proper namer
```

Comment:
let's keep "client_name" here, and we probably want to remove the FIXME now

Reply:
switched to be module_name and class_name